models/template.go

```diff
@@ -22,12 +22,16 @@ const (
 	TemplateNameSignupClinic          TemplateName = "signup_clinic_confirmation"
 	TemplateNameSignupCustodial       TemplateName = "signup_custodial_confirmation"
 	TemplateNameSignupCustodialClinic TemplateName = "signup_custodial_clinic_confirmation"
+	TemplateNameTestTemplate          TemplateName = "test_template"
```

Comment:
Do we need to have this test property here? Looks like it's only used by templates/test.go

Reply:
the idea was to use the same code mechanism as for the other templates creation. Anyhow I will remove it